package.json

```diff
@@ -66,6 +66,7 @@
     "grunt-sass": "^2.0.0",
     "grunt-text-replace": "^0.4.0",
     "grunt-xmlmin": "^0.1.8",
+    "jasmine-reporters": "^2.2.1",
```

Comment:
Do we really want to use jasmine and mocha together in the same project?  Their syntax is almost identical, and we're using mocha already in this project, and also using it in multiple other projects.